WooCommerce/Classes/ViewRelated/Dashboard/MyStore/PeriodDataViewController.swift

```diff
@@ -38,8 +38,27 @@ class PeriodDataViewController: UIViewController {
     @IBOutlet private weak var chartAccessibilityView: UIView!
 
     private var lastUpdatedDate: Date?
-    private var yAxisMinimum: String = Constants.chartYAxisMinimum.humanReadableString()
-    private var yAxisMaximum: String = ""
+
+    private var revenueItems: [Double] {
```

Comment:
Should we name it as `grossSales` instead? I feel like that's easier to understand. 🤔